src/bosh-monitor/lib/bosh/monitor/resurrection_manager.rb

```diff
@@ -0,0 +1,108 @@
+module Bosh::Monitor
+  class ResurrectionManager
+    def initialize()
+      @parsed_rules = []
+      @logger = Bhm.logger
+      @active_ids = []
+    end
+
+    def resurrection_enabled?(deployment_name, instance_group)
+      enabled = true
+      @parsed_rules.each do |parsed_rule|
+        enabled = enabled && parsed_rule.enabled? if parsed_rule.applies?(deployment_name, instance_group)
+      end
+
+      return enabled
+    end
+
+    def update_rules(resurrection_configs)
+      new_parsed_rules = []
+      return if resurrection_configs.nil? || resurrection_configs.empty?
+
+      ids = resurrection_configs.map {|resurrection_config| resurrection_config['id']}
+      if @active_ids.to_set != ids.to_set
```

Comment:
Rather than keeping track of the `@active_ids`, can we keep track of the sha256 of the content of active configs? This can be done through `Digest::SHA256.digest(resurrection_config['content'])`.